net/stubby/files/stubby.conf

```diff
@@ -19,48 +19,48 @@ config stubby 'global'
        list listen_address '0::1@5453'
        # option log_level '7'
        # option command_line_arguments ''
-       # option tls_cipher_list 'EECDH+AESGCM:EECDH+CHACHA20'
-       # option tls_ciphersuites 'TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256'
+       option tls_cipher_list 'EECDH+CHACHA20:EECDH+AESGCM'
+       option tls_ciphersuites 'TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384'
```

Comment:
This was enabled. Is it done on purpose?

Reply:
Yes I intentionally enabled them. Stubby preferred AES over ChaCha by default, which might not be the best choice to most embedded devices. I have to enable these two lines in config file to set it to prefer ChaCha over AES.